packages/suite/src/views/wallet/coinmarket/buy/components/BuyForm/Footer/index.tsx

```diff
@@ -126,7 +126,9 @@ const Footer = () => {
                                     );
                                 }}
                                 isClearable={false}
-                                minWidth="160px"
+                                width={190}
```

Comment:
I think we should keep the original `minWidth` here
<img width="438" alt="Screenshot 2020-12-02 at 14 18 35" src="https://user-images.githubusercontent.com/6961901/100877597-48c63a00-34a9-11eb-87e1-b48170e83083.png">